db/schema.rb

```diff
@@ -589,6 +591,21 @@
     t.index ["user_id"], name: "index_page_views_on_user_id"
   end
 
+  create_table "pages", force: :cascade do |t|
```

Comment:
Don't think this `create_table` should be included in this PR right?

Reply:
Yeah, working on multiple PRs, got my schema messed. Fixed.